.travis.yml

```diff
@@ -1,54 +1,20 @@
 sudo: false
 language: java
 
-before_script:
-  - test "x$RUN_CHECKSTYLE" != 'x' || ant -Djava.awt.headless=true download_jars install
-  - test "x$RUN_CHECKSTYLE" != 'xtrue' || ant -Djava.awt.headless=true download_checkstyle
-
-before_install:
-    - wget --no-check-certificate https://www.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.tar.gz
-    - tar -xzvf apache-ant-1.10.5-bin.tar.gz
-    - export PATH="$(pwd)/apache-ant-1.10.5/bin:$PATH"
-    - echo $(ant -version)
-
 # skip default "install" command
 install: true
 
-env:
-  global:
-  - ANT_OPTS="-Ddisable-svnCheck=true -Djava.awt.headless=true -Drmi_force_localhost=true -Dskip.bug52310=true"
+before_cache:
+  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+cache:
+  directories:
+    - $HOME/.gradle/caches/
+    - $HOME/.gradle/wrapper/
 
 matrix:
   include:
-    - jdk: oraclejdk8
-      env: RUN_CHECKSTYLE=true
-      script: ant -Djava.awt.headless=true checkstyle
-    - name: Coverage on our default build with Oracle Java 8
-      jdk: oraclejdk8
-      script:
-          - ant coverage-travis
-      after_success:
-          - bash <(curl -s https://codecov.io/bash)
-      sudo: true # otherwise TEST_HTTPS.jmx -> analytics.usa.gov does not work
     - name: Tests with OpenJDK 8
-      jdk: openjdk8
-      script:
-          - ant test
-    - name: Tests with OpenJDK 11
-      jdk: openjdk11
-      script:
-          - ant test
-      sudo: true # otherwise TEST_HTTPS.jmx -> analytics.usa.gov does not work
-    - name: Tests with OpenJDK EA
-      jdk: openjdk-ea
+      jdk: oraclejdk8
```

Comment:
Could we re-add JDK 11 and EA?

Reply:
Makes no sense for now. I want fast Travis feedback, so I removed all the extra Travis configs.

Final version would include all the relevant JDKs.